.github/workflows/build-ffmpeg.yml

```diff
@@ -164,62 +242,133 @@ jobs:
       - name: libx264
         env:
           root_path: ${{ steps.root.outputs.root_path }}
+          CCPREFIX: ${{ steps.cross.outputs.CCPREFIX }}
         working-directory: ffmpeg_sources/x264
         run: |
+          echo "::group::configure options"
+          ./configure --help || true  # this command has a non zero exit status, but we should continue anyway
+          echo "::endgroup::"
+
+          echo "::group::configure extra flags for cross compilation"
+          extra_configure=$(cat <<VAREOF
+          --host=${{ matrix.arch }}-${{ matrix.os_type }}
+          --cross-prefix=${CCPREFIX}
+          VAREOF
+          )
+          if [[ ${{ matrix.arch }} != "aarch64" ]]; then
+            extra_configure=""
+          fi
+          echo "$extra_configure"
+          echo "::endgroup::"
+
+          echo "::group::configure"
           PATH="$root_path/bin:$PATH" \
             PKG_CONFIG_PATH="$root_path/ffmpeg_build/lib/pkgconfig" \
             ./configure \
+            $extra_configure \
             --prefix="$root_path/ffmpeg_build" \
             --bindir="$root_path/bin" \
             --disable-cli \
             --enable-static
+          echo "::endgroup::"
+
+          echo "::group::make"
           PATH="$root_path/bin:$PATH" make -j$(nproc || sysctl -n hw.logicalcpu)
+          echo "::endgroup::"
+
+          echo "::group::make install"
           make install
+          echo "::endgroup::"
 
       - name: libx265
         env:
           root_path: ${{ steps.root.outputs.root_path }}
+          CCPREFIX: ${{ steps.cross.outputs.CCPREFIX }}
         working-directory: ffmpeg_sources/x265_git
         run: |
+          echo "::group::configure extra flags for cross compilation"
+          extra_configure="-DCMAKE_TOOLCHAIN_FILE=build/${{ matrix.arch }}-${{ matrix.os_type }}/crosscompile.cmake"
+          cat ./build/${{ matrix.arch }}-${{ matrix.os_type }}/crosscompile.cmake || true
+          if [[ ${{ matrix.arch }} != "aarch64" ]]; then
+            extra_configure="-DENABLE_HDR10_PLUS=1"
```

Comment:
Todo: will this build aarch64?

Reply:
`aarch64` is not currently supported: https://bitbucket.org/multicoreware/x265_git/src/cc25ead66daa466ab27cc14897a48a1b61bcf64e/source/dynamicHDR10/CMakeLists.txt#lines-45:46